features/src/good.cpp

```diff
@@ -42,9 +42,8 @@
 #include <pcl/impl/instantiate.hpp>
 // Instantiations of specific point types
 #ifdef PCL_ONLY_CORE_POINT_TYPES
-  PCL_INSTANTIATE_PRODUCT(GOODEstimation, ((pcl::PointXYZ)(pcl::PointXYZI)(pcl::PointXYZRGB)(pcl::PointXYZRGBA)))
+PCL_INSTANTIATE_PRODUCT(GOODEstimation, ((pcl::PointXYZ)(pcl::PointXYZI)(pcl::PointXYZRGB)(pcl::PointXYZRGBA))((pcl::Histogram<3*5*5>)))
```

Comment:
Here you need to put the values of template parameters for which we want to instantiate. In our case, the second parameter in just the bin size. So, this should be simply `5`, not `pcl::Histogram<3*5*5>`.